make/hotspot/lib/JvmFeatures.gmk

```diff
@@ -174,6 +174,12 @@ ifeq ($(call check-jvm-feature, link-time-opt), true)
         -fno-fat-lto-objects
     JVM_LDFLAGS_FEATURES += $(CXX_O_FLAG_HIGHEST_JVM) -flto=auto \
         -fuse-linker-plugin -fno-strict-aliasing
+  else ifeq ($(call isCompiler, clang), true)
+    JVM_CFLAGS_FEATURES += -flto -fno-strict-aliasing
+    ifeq ($(call isBuildOs, aix), true)
+      JVM_CFLAGS_FEATURES += -ffat-lto-objects
+    endif
+    JVM_LDFLAGS_FEATURES += $(CXX_O_FLAG_HIGHEST_JVM) -flto -fno-strict-aliasing
```

Comment:
I saw this too late, shouldn't it be isTargetOs instead of isBuildOs? Or am I mistaken and only gtest is affected, hence isBuildOs? Actually I'm not sure which one would be appropriate for gtest

Reply:
I think in the AIX case the target an build OS is always the same so it should be okay as it is.
isTargetOs might generally fit better.

Reply:
So currently we do not allow for cross-OS compilation. (This might change in the future.) So in practice, target OS and build OS is the same, and we might have been sloppy elsewhere. But you are correct Julian that this would have been better.